package.json

```diff
@@ -68,6 +68,7 @@
     "test": "node ./out/test/runTest.js"
   },
   "devDependencies": {
+    "@tootallnate/once": "^1.1.2",
```

Comment:
Are all the new packages needed?

Reply:
it wouldnt build on my mac without them

Reply:
i dont know if you have them added globally?

Reply:
For me it's working without them. @JamesWoolfenden can you please double verify?

Reply:
removed.